src/components/MessageLogger/MessageList.tsx

```diff
@@ -0,0 +1,28 @@
+import React, { useEffect, useState } from "react";
+import { LineMessage } from "@components/MessageLogger/LineMessage";
+import styles from "@components/MessageLogger/MessageList.module.scss";
+import { MessageCounter } from "@adapters/Message";
+
+interface Props {
+    messages: MessageCounter[]
+}
+
+export const MessageList = ({ messages }: Props) => {
+
+    return (
+        <div id={styles.containerMessages}>
+            <ul className={styles.lineMsgUl}>{messages.map((item, index) => {
+                return (
+                    <LineMessage key={index} message={item.msg} count={item.count} />
```

Comment:
It is not recommended to use the index as a key (https://robinpokorny.com/blog/index-as-a-key-is-an-anti-pattern/). You should add an id property to the MessageCounter element. The id can be generated with nanoid. Its used somewhere in the project, look for it and it should be clear.